setup.py

```diff
@@ -77,12 +78,8 @@ def run(self):
         if os.path.exists(self.source_server_file):
             os.remove(self.source_server_file)
 
-        # Remove build/lib directory.
-        if os.path.exists('build/lib/'):
```

Comment:
Whats the reason for removing this?

Reply:
When we use `assemble` the build directory is never generated.